config/locales/exercises/en.yml

```diff
@@ -4,8 +4,11 @@ en:
       welcome: "Welcome to %{exercise_title} on Exercism's %{track_title} Track."
       help_reference: "If you need help running the tests or submitting your code, check out `HELP.md`."
       hints_reference: "If you get stuck on the exercise, check out `HINTS.md`, but try and solve it without using those first :)"
-      tests_intro: "Running the tests"
-      submitting_solution_intro: "Submitting your solution"
+      tests_header: "Running the tests"
+      submitting_solution_header: "Submitting your solution"
       submitting_solution: "You can submit your solution using the `exercism submit %{solution_file_paths}` command.\nThis command will upload your solution to the Exercism website and print the solution page's URL.\n\nIt's possible to submit an incomplete solution which allows you to:\n\n- See how others have completed the exercise\n- Request help from a mentor"
-      help_intro: "Need to get help?"
-      help: "If you'd like help solving the exercise, check the following pages:\n\n- The [%{track_title} track's documentation](https://exercism.org/docs/tracks/%{track_slug})\n- [Exercism's programming category on the forum](https://forum.exercism.org/c/programming/5)\n- The [Frequently Asked Questions](https://exercism.org/docs/using/faqs)\n\nShould those resources not suffice, you could submit your (incomplete) solution to request mentoring."
+      help_header: "Need to get help?"
+      help_intro: "If you'd like help solving the exercise, check the following pages:"
+      help_check_pages: "Should those resources not suffice, check the following pages:"
+      help_pages: "- The [%{track_title} track's documentation](https://exercism.org/docs/tracks/%{track_slug})\n- The [%{track_title} track's programming category on the forum](https://forum.exercism.org/c/programming/%{track_slug})\n- [Exercism's programming category on the forum](https://forum.exercism.org/c/programming/5)\n- The [Frequently Asked Questions](https://exercism.org/docs/using/faqs)"
```

Comment:
It gets lost in this diff, but I've added an additional resource to link to for help: the track's forum category. You can see what that looks like in some of t the tests that are modified in this PR